environment.yml

```diff
@@ -18,3 +18,4 @@ dependencies:
   - xarray
   - geogif
   - zarr
+  - pre-commit
```

Comment:
This adds `pre-commit` to the default cluster environment, which isn't strictly necessary. That said, `pre-commit` is small and I don't recall running into any solve issues with it around in the other environments.